services/sasquatch/values.yaml

```diff
@@ -0,0 +1,100 @@
+# Default values for Sasquatch.
```

Comment:
This is missing a lot of helm-docs comments, and it would be nice to generate a `README.md` for the services/sasquatch directory as well in this new style.